Quadriga/src/main/java/edu/asu/spring/quadriga/service/impl/transformation/TransformationManager.java

```diff
@@ -22,18 +25,26 @@
 
     @Autowired
     private ITransformFilesDAO transformationDAO;
+    
+    @Autowired
+    private ITransformationSaveService transformationFileService;
 
     @Transactional
     @Override
-    public void saveTransformation(String title, String description,
-            String patternFileName, String patternTitle,
-            String patternDescription, String mappingFileName,
-            String mappingTitle, String mappingDescription,String userName) {
-        TransformFilesDTO tranformDTO = new TransformFilesDTO(title,
-                description, patternFileName, patternTitle, patternDescription,
-                mappingFileName, mappingTitle, mappingDescription, userName);
+    public void saveTransformations(ITransformationFile transformationFile){
+        TransformFilesDTO tranformDTO = new TransformFilesDTO(transformationFile.getTitle(),
+                transformationFile.getDescription(), transformationFile.getPatternFileName(), transformationFile.getPatternTitle(), 
+                transformationFile.getPatternDescription(), transformationFile.getMappingFileName(), transformationFile.getMappingTitle(), 
+                transformationFile.getMappingDescription(), transformationFile.getUserName());
         tranformDTO.setId(transformationDAO.generateUniqueID());
         transformationDAO.saveNewDTO(tranformDTO);
+        
+        try{
+        transformationFileService.saveFileToLocal(transformationFile);
```

Comment:
@jdamerow @yyk16293 I came across a feature in eclipse that automatically formats the code and also organizes imports when we save the code. In addition it also formats the code that we have changed. I feel it would be really helpful for us, because we always forget to format or sometime we format the entire code.

http://help.eclipse.org/mars/index.jsp?topic=%2Forg.eclipse.jdt.doc.user%2Freference%2Fpreferences%2Fjava%2Feditor%2Fref-preferences-save-actions.htm